src/index.css

```diff
@@ -1,31 +1,137 @@
 body {
-  margin: 10px;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  margin: 0;
+  background: #f8f8f8;
+}
+
+.navbar {
+  height: 5.938rem;
+  background: white;
+  padding-top: 30px;
+  padding-left: 30px;
+}
+
+.navlink {
+  text-decoration: none;
+  margin-left: 20px;
+}
+
+#logo {
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.nav {
+  width: 3.688rem;
+  height: 1rem;
+  font-family: Montserrat, sans-serif;
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.navbar__container {
```

Comment:
- [ ] Please double check the font and colors even the layout of the original design , like your user icon and also the alignment of your logo and books and categories are not on the same line

Original design 👇 

![image](https://user-images.githubusercontent.com/75126481/179217845-03d38118-d3bf-4335-aca5-bdb4fc1dc7ff.png)

Your design 👇 

![image](https://user-images.githubusercontent.com/75126481/179217958-5c03f3cf-df1a-4f58-b28e-ce57fb784e30.png)